resilience4j-timelimiter/src/main/java/io/github/resilience4j/timelimiter/utils/MetricNames.java

```diff
@@ -0,0 +1,8 @@
+package io.github.resilience4j.timelimiter.utils;
+
+public class MetricNames {
+    public static final String DEFAULT_PREFIX = "resilience4j.timelimiter";
+    public static final String SUCCESSFUL_CALLS = "number_of_successful_calls";
```

Comment:
Can we please use the same names like the CircuitBreaker:

 public static final String SUCCESSFUL = "successful";
 public static final String FAILED = "failed";
 public static final String TIMEOUT = "timeout";